packages/paste-token-contrast-checker/jestTransformer.js

```diff
@@ -0,0 +1,10 @@
+import babel from '@babel/core';
+
+module.exports = {
```

Comment:
Why do we need this here?

Reply:
Default create-react-app bootstrapping. Can probably delete some of these files actually